modules/ossm-cr-gateway.adoc

```diff
@@ -5,13 +5,7 @@
 [id="ossm-cr-gateway_{context}"]
 = Istio gateway example
 
-Here is an example that illustrates the Istio gateway parameters for the {ProductName} custom resource and a description of the available parameters with appropriate values.
-
-[WARNING]
-====
-Automatic route creation does not currently work with multi-tenancy. Set `ior_enabled` to `false` for multi-tenant installations.
-====
-
+Here is an example that illustrates the Istio gateway parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
 
 [source,yaml]
 ----
```

Comment:
Example and the table below contain `ior_enabled`. Didn't we remove ior support?

Reply:
Also, indentation in the example looks inconsistent.

Reply:
The indentation is correct here: https://servicemesh--ocpdocs.netlify.com/openshift-enterprise/latest/service_mesh/service_mesh_install/customizing-installation-ossm.html#ossm-cr-gateway_customizing-installation-ossm